src/06-loops.js

```diff
@@ -61,7 +80,25 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+function containsWhileLoop(array, target) {
+  let i = 0;
+  while(i < array.length){
+    if(array[i] === target){
+      return true;
+    }
+    i++
```

Comment:
Let's clean up our spacing here and remove our commented out code.  A good habit to do before pushing is remove all  ` console.log ` or commented out code.  We can leave comments but not unused code.